modules/standard/Math.chpl

```diff
@@ -367,11 +367,12 @@ module Math {
   }
 
 
-  /* Returns the complex conjugate of the argument `z`.
+  /* Returns the complex conjugate of the complex argument `z`.
 
      :rtype: A complex number of the same type as `z`.
   */
-  inline proc conjg(z: complex(?w)): complex(w) {
+  inline proc conjg(z: complex(?w)) where isComplexType(z.type): complex(w)
```

Comment:
I was too. It's necessary because of the explicit `: complex(?w)` type. Other numeric types get coerced to complex resulting in this overload taking priority over the other more general overload. I just made a commit that drops the explicit type so that all 3 overloads consistently use `where` clauses to resolve distinctly.